modules/tinymce/src/plugins/fullscreen/test/ts/webdriver/FullscreenPluginNativeModeTest.ts

```diff
@@ -0,0 +1,68 @@
+import { UnitTest } from '@ephox/bedrock-client';
+
+import { Editor as McEditor } from '@ephox/mcagar';
+import FullscreenPlugin from 'tinymce/plugins/fullscreen/Plugin';
+import SilverTheme from 'tinymce/themes/silver/Theme';
+import { Pipeline, Log, NamedChain, Chain, UiFinder, RealMouse, Waiter, Step } from '@ephox/agar';
+
+UnitTest.asynctest('browser.tinymce.plugins.fullscreen.FullScreenPluginNativeModeTest', (success, failure) => {
+  FullscreenPlugin();
+  SilverTheme();
+
+  const cSetupEditor = McEditor.cFromSettings({
+    plugins: 'fullscreen',
+    toolbar: 'fullscreen',
+    theme: 'silver',
+    base_url: '/project/tinymce/js/tinymce',
+    fullscreen_native: true
+  });
+
+  const getFullscreenElement = () => {
```

Comment:
If the fullscreen logic was moved to a separate module, you could maybe reuse the logic here instead of duplicating it?

Reply:
Will do. Do you think we need any of it in Sugar?

Reply:
I'm honestly not sure, so go with whatever you feel it best? 🤷

Reply:
It's probably pretty specific to the fullscreen plugin so I'll leave it there for now. I guess if we discover that the logic is useful elsewhere we can move it.